max_training_framework/wml_setup.py

```diff
@@ -261,6 +262,17 @@ def env_extract(access_key, secret_access_key, apikey,
     for env_var in ['ML_ENV', 'ML_APIKEY', 'ML_INSTANCE']:
         if os.environ.get(env_var) is None:
             wml_env_check_flag = 'N'
+
+        if env_var == 'ML_ENV':
+            os.environ.get(env_var)
+            h = urlparse(os.environ.get(env_var)).hostname
+            if len([sh for sh in ['us-south', 'eu-gb']
+                    if h.startswith(sh)]) == 0:
+                print('Error. WML service location "{}" does not '
```

Comment:
The error message should include the name of the problematic environment variable and describe what the program is doing, e.g.:
```
Error. Environment variable ML_ENV is set to "eu-de.ml.cloud.ibm.com". The Watson Machine Learning service in this region does not provide the required training functionality. The existing WML environment variables will be ignored.
```